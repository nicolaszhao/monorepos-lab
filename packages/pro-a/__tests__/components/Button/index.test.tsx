import React from 'react';
import { render } from '@testing-library/react';
import Button from '../../../src/components/Button';

describe('test Button', () => {
  test('test button snapshot', () => {
    const { container } = render(<Button>XXX</Button>);
    expect(container).toMatchSnapshot();
  });
});
