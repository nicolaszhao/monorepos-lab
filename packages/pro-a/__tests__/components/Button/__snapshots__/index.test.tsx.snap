// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Button test button snapshot 1`] = `
<div>
  <button>
    XXX
  </button>
</div>
`;
